### PR TITLE
Iframe clean

### PR DESCRIPTION
**Pull request by xmontana**
Format text summary and fullsummary allowing trusted text like <iframe> if enabletrusttext is enabled. Related to issue https://github.com/WisdmLabs/moodle-format_remuiformat/issues/87

### CODE REVIEW

**Comment by institut-agro-ead** [comment]
The use of format_text for summary and fullcontent has been added with v4.1.4 but seems to be useless ?
format_text is already called in moodle core via `$this->courseformatdatacommontrait->course_section_cm_text` or `$renderer->format_summary_text` meaning the module content is already formatted once (with noclean option set to true) then the result is once again formatted without the `noclean` option. Iframes disappear, so H5Ps embeded inside labels via the H5P button inside text editor don’t show.
Basically, I think those lines should simply be removed as they weren’t here before v4.1.4 and seem to be redundant.
